android/capacitor/src/main/assets/native-bridge.js

```diff
@@ -147,6 +148,7 @@ var nativeBridge = (function (exports) {
             if (doc) {
                 const docAddEventListener = doc.addEventListener;
                 doc.addEventListener = (...args) => {
+                    var _a;
```

Comment:
shouldn't this (and others) be a let?

Reply:
This is a generated file - I'm not certain where this comes from.